packages/components/src/AppGuidedTour/AppGuidedTour.test.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { mount } from 'enzyme';
+import AppGuidedTour from './AppGuidedTour.component';
+import { LOADING_STEP_STATUSES } from '../Stepper';
+
+const DEFAULT_PROPS = {
+	appName: 'app name',
+	isOpen: true,
+	steps: [],
+	demoContentSteps: [],
+	onImportDemoContent: () => {},
```

Comment:
Can you check callbacks too?

Reply:
what do you mean?

Reply:
using jest.fn and be sure that there are called 😅

Reply:
I do ;)

Reply:
I guess you meant it for onRequestClose, I'll add a test